tokens/movistar-classic.json

```diff
@@ -145,15 +155,25 @@
       "type": "color",
       "description": "movistarBlue30"
     },
+    "buttonSecondaryBackgroundHoverInverse": {
+      "value": "rgba({palette.white}, 0.2)",
+      "type": "color",
+      "description": "white"
+    },
+    "buttonSecondaryBackgroundSelectedInverse": {
+      "value": "rgba({palette.white}, 0.2)",
+      "type": "color",
+      "description": "white"
+    },
     "textButtonPrimary": {
       "value": "{palette.white}",
       "type": "color",
       "description": "white"
     },
     "textButtonPrimaryInverse": {
-      "value": "{palette.movistarBlue}",
+      "value": "{palette.white}",
```

Comment:
why did you change this?

if you use white with buttonPrimaryBackgroundInverse the button will be completly white!
```
 "buttonPrimaryBackgroundInverse": {
      "value": "{palette.white}",
      "type": "color",
      "description": "white"
    },
```

remember, movistar-classic have to work same as today, just include the new states for this skin